File: Modul-2/Soal-2/PRAK202-2410817310012-FAISALTANJUNG.py
nilai1 = float(input("Masukkan Nilai Pertama: "))
nilai2 = float(input("Masukkan Nilai Kedua: "))

hasil = nilai1 + nilai2


if nilai1.is_integer():
    nilai1_output = f"{int(nilai1)}"
else:
    nilai1_output = f"{nilai1:.2f}"


print(f"Hasil dari penjumlahan nilai pertama \"{nilai1_output}\" dan nilai kedua \"{nilai2}\" adalah \"{hasil:.2f}\"\n")

File: Modul-2/Soal-2/PRAK202-2410817310012-FAISALTANJUNG.c
#include <stdio.h>
int main(){
    float angka1, angka2, hasil;

    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &angka1);
    printf("Masukkan Nilai Kedua : ");
    scanf("%f", &angka2);

    hasil = angka1 + angka2;

    printf("Hasil dari penjumlahan Nilai Pertama ");
if(angka1 == (int)angka1)
    printf("\"%d\" ", (int)angka1);
else
    printf("\"%.2f\" ", angka1);
    printf("dan nilai Kedua \"%.1f\" adalah \"%.2f\"\n", angka2, hasil);
    return 0;
}